eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -140,6 +133,10 @@ jobs:
         parameters:
           SourceDirectory: $(Build.SourcesDirectory)
 
+      - template: ../../../common/pipelines/templates/steps/verify-links.yml
+        parameters:
+          ServiceDirectory: ${{parameters.ServiceDirectory}}
```

Comment:
This is where I was expecting you to pass `sdk/${{ parameters.ServiceDirectory }}`